prestodb/hive3.1-hive/Dockerfile

```diff
@@ -10,7 +10,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM prestodb/centos7-oj8:unlabelled
+FROM prestodb/centos7-oj8:latest
```

Comment:
Same

Reply:
The "unlabelled" tag is required to go through Makefile build process. I added latest since we are building manually. Should I revert this to unlabelled?

Reply:
Hmm OK, well there is no great answer, but yeah lets go with latest and assume next time the Makefile is used.